### PR TITLE
Feat/basic starter

### PR DESCRIPTION
**Pull request by logan-anderson**
Adds a basic tinacms starter for folks to take a look at and play around with.

### CODE REVIEW

File: examples/basic/pages/posts/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  })
+
+  return {
+    props: {
+      ...tinaProps,
+    },
+  }
+}
+
+// export const getStaticProps = async () => {
```
Comment (logan-anderson) [inline examples/basic/pages/posts/index.js:46]: This should be deleted

File: examples/basic/pages/admin/[[...slug]].js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { TinaAdmin } from 'tinacms'
```
Comment (logan-anderson) [inline examples/basic/pages/admin/[[...slug]].js:1]: Is it OK to use the Admin right now?

Comment (Enigmatical) [reply examples/basic/pages/admin/[[...slug]].js:1]: I don't see why not; I assume it is working and has the right version of TinaCMS.

This is more a question for @jeffsee55 and the product team on when they want `TinaAdmin` to start being bundled with our starters.

Comment (logan-anderson) [reply examples/basic/pages/admin/[[...slug]].js:1]: Talked about it, We are keeping admin in.

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**